### PR TITLE
[FR] Update `Offset` `Local_offset` `Online_offset` `Universal_offset`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Offset/Local_offset/fr.md [hunk 1]
```diff
@@ -23,3 +37,7 @@ Si un local offset est défini, osu! affichera le local offset dans l'interface
 osu! vous informera également du local offset avant de commencer le jeu.
 
 ![Notification de local offset](img/local-offset-notice.png "Notification de local offset")
+
+## Notes et références
+
+[^hit-difference]: le décalage requis pouvant être déduit soit des temps sur le compteur de score atterrissant principalement au même endroit non centré, soit des valeurs cohérentes de [hit error](/wiki/Gameplay/Accuracy#error) sur l'[écran des résultats](/wiki/Client/Interface#écran-des-résultats).
```
Comment (Ganondorf-1) [inline wiki/Offset/Local_offset/fr.md:43]:
```suggestion
[^hit-difference]: Le décalage requis pouvant être déduit soit des temps sur le compteur de score atterrissant principalement au même endroit non centré, soit des valeurs cohérentes de [hit error](/wiki/Gameplay/Accuracy#error) sur l'[écran des résultats](/wiki/Client/Interface#écran-des-résultats).
```

File: wiki/Offset/Online_offset/fr.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
 ---
 tags:
   - online song offset
-outdated_translation: true
-outdated_since: c982b3d23c9577d6d0353a006ca6528771511319
+  - décalage de musique en ligne
```
Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:4]:
```suggestion
```
Not useful imo

Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:4]:
```suggestion
  - décalage en ligne
```

File: wiki/Offset/Online_offset/fr.md [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
 ---
 
 # Online offset
 
 *Pour d'autres utilisations, voir [Offset](/wiki/Offset).*
 
-L'**online offset** est un décalage qui peut être appliqué à une [beatmap](/wiki/Beatmap) après son [classement](/wiki/Beatmap/Category#classée) initial. C'est généralement le cas lorsqu'une beatmap a besoin de petits changements de timing pour s'aligner correctement avec la musique. Les valeurs d'online offset sont appliquées en plus des valeurs de l'[offset universel](/wiki/Offset/Universal_offset) et de l'[offset local](/wiki/Offset/Local_offset) d'un joueur.
+L'**online offset** (ou décalage en ligne) est un décalage qui peut être appliqué à une [beatmap](/wiki/Beatmap) après son [classement](/wiki/Beatmap/Category#classée) initial. C'est généralement le cas lorsqu'une beatmap a besoin de petits changements de [timing](/wiki/Beatmapping/Timing) pour s'aligner correctement avec la musique. Les valeurs d'online offset sont appliquées en plus des valeurs de l'[offset universel](/wiki/Offset/Universal_offset) et de l'[offset local](/wiki/Offset/Local_offset) d'un joueur.
```
Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:11]:
```suggestion
L'**online offset** (ou décalage en ligne) est un décalage qui peut être appliqué à une [beatmap](/wiki/Beatmap) après son [classement](/wiki/Beatmap/Category#classée) initial. C'est généralement le cas lorsqu'une beatmap a besoin de petits changements de [timing](/wiki/Beatmapping/Timing) pour s'aligner correctement avec la musique. Les valeurs d'online offset sont appliquées en plus des valeurs de l'[offset universel](/wiki/Offset/Universal_offset) et de l'[offset local](/wiki/Offset/Local_offset) du joueur.
```

File: wiki/Offset/Online_offset/fr.md [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
+
+## Comportement
+
+L'online offset est personnalisé pour chaque beatmap mal synchronisé par un administrateur du site web ou par un membre de l'équipe de la [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team), et est récupéré de manière automatisée par osu! en même temps que les [classements](/wiki/Ranking). De la même manière que le décalage local, il fonctionne en décalant tous les éléments de gameplay par rapport à la piste audio d'une quantité spécifiée de millisecondes :
```
Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:15]:
```suggestion
L'online offset est personnalisé par un administrateur du site web ou par un membre de la [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) pour chaque beatmap mal synchronisée, et est récupéré de manière automatisée par osu! en même temps que les [classements](/wiki/Ranking). De la même manière que le décalage local, il fonctionne en décalant tous les éléments de gameplay par rapport à la piste audio d'une quantité spécifiée de millisecondes :
```

File: wiki/Offset/Online_offset/fr.md [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
+
+- Les valeurs **Négatives** déplacent les éléments de gameplay **plus tôt**.
```
Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:17]:
```suggestion
- Les valeurs **négatives** déplacent les éléments de gameplay **plus tôt**.
```

File: wiki/Offset/Online_offset/fr.md [hunk 6]
```diff
@@ -1,15 +1,21 @@
...
+- Les valeurs **Positives** déplacent les éléments de gameplay **plus tard**.
```
Comment (Ganondorf-1) [inline wiki/Offset/Online_offset/fr.md:18]:
```suggestion
- Les valeurs **positives** déplacent les éléments de gameplay **plus tard**.
```

File: wiki/Offset/Universal_offset/fr.md [hunk 7]
```diff
@@ -1,13 +1,32 @@
...
 
-La valeur optimale du décalage universel de chaque joueur diffère en raison des variations de leurs systèmes. Un décalage universel, autre que `0`, ne devrait être utilisé que lorsque **tous les beatmaps** ont un retard notable. Sinon, un [offset local](/wiki/Offset/Local_offset) devrait être utilisé à la place. Un décalage universel mal déterminé causera d'importants problèmes de timing sur toutes les beatmaps.
+## Comportement
+
+L'universal offset fonctionne en ajoutant un délai spécifié entre la chanson de chaque beatmap et les autres sons et graphiques associés. Contrairement à l'offset [local](/wiki/Offset/Local_offset) ou [en ligne](/wiki/Offset/Online_offset), il est appliqué à l'audio plutôt qu'aux éléments de gameplay, produisant l'effet inverse :
```
Comment (Ganondorf-1) [inline wiki/Offset/Universal_offset/fr.md:17]:
```suggestion
L'universal offset fonctionne en ajoutant un délai spécifié entre la musique de chaque beatmap et les autres sons et graphiques associés. Contrairement à l'offset [local](/wiki/Offset/Local_offset) ou [en ligne](/wiki/Offset/Online_offset), il est appliqué à l'audio plutôt qu'aux éléments de gameplay, produisant l'effet inverse :
```

File: wiki/Offset/Universal_offset/fr.md [hunk 8]
```diff
@@ -1,13 +1,32 @@
...
+
+- Les valeurs **Positives** déplacent les éléments de gameplay **plus tôt**.
```
Comment (Ganondorf-1) [inline wiki/Offset/Universal_offset/fr.md:19]:
```suggestion
- Les valeurs **positives** déplacent les éléments de gameplay **plus tôt**.
```

File: wiki/Offset/Universal_offset/fr.md [hunk 9]
```diff
@@ -1,13 +1,32 @@
...
+- Les valeurs **Négatives** déplacent les éléments de gameplay **plus tard**.
```
Comment (Ganondorf-1) [inline wiki/Offset/Universal_offset/fr.md:20]:
```suggestion
- Les valeurs **négatives** déplacent les éléments de gameplay **plus tard**.
```

File: wiki/Offset/fr.md [hunk 10]
```diff
@@ -1,22 +1,27 @@
...
-outdated_translation: true
-outdated_since: 87257df3878d4da3af9de4d36ddd84a23e478a10
----
-
 # Offset
 
-Le terme **Offset** peut avoir plusieurs significations :
+Un **offset** est un paramètre qui décale et aligne temporellement différents éléments d'une [beatmap](/wiki/Beatmap) comme la piste audio, les [hitsounds](/wiki/Beatmapping/Hitsound), les [objets](/wiki/Gameplay/Hit_object), ou la vidéo de fond. Il existe différents types de décalages dans le jeu qui affectent différents éléments.
+
+Selon le type de décalage, il est défini par le créateur de la beatmap, personnalisé par le joueur ou ajusté par l'un des administrateurs du site. La plupart des décalages sont appliqués par beatmap à l'exception du [universal offset](#gameplay).
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:5]:
```suggestion
Selon le type de décalage, il est défini par le créateur de la beatmap, personnalisé par le joueur ou ajusté par l'un des administrateurs du site. La plupart des décalages sont appliqués par beatmap à l'exception de l'[universal offset](#gameplay).
```

File: wiki/Offset/fr.md [hunk 11]
```diff
@@ -1,22 +1,27 @@
...
+
+## Types
+
+### Gameplay
+
+- [Local offset](/wiki/Offset/Local_offset), un paramètre par beatmap qui décale tous les éléments du jeu par rapport à la piste audio.
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:11]:
```suggestion
- [Local offset](/wiki/Offset/Local_offset), un paramètre par beatmap qui décale tous les éléments de gameplay par rapport à la piste audio.
```

File: wiki/Offset/fr.md [hunk 12]
```diff
@@ -1,22 +1,27 @@
...
+- [Online offset](/wiki/Offset/Online_offset), un changement de timing appliqué par la [NAT](/wiki/People/Nomination_Assessment_Team) aux beatmaps [classées](/wiki/Beatmap/Category#classée) avec un décalage de beatmap incorrect afin de l'ajuster.
+- [Universal offset](/wiki/Offset/Universal_offset), un paramètre qui affecte globalement le timing des beatmaps en décalant la piste audio par rapport à tous les autres éléments du jeu.
+
+### Mapping
 
-## Paramètres de timing
+- [Points de timing non hérités](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) ("ligne rouge"), le début d'une nouvelle [timing section](/wiki/Beatmapping/Timing_section). Les points de synchronisation se situent à un décalage par rapport au début du fichier audio, mais le point de synchronisation lui même est parfois appelé "décalage".
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:17]:
```suggestion
- [Points de timing non hérités](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) ("ligne rouge"), le début d'une nouvelle [timing section](/wiki/Beatmapping/Timing_section). Les points de timing se situent à un décalage par rapport au début du fichier audio, mais ce point de timing est parfois appelé "décalage".
```

File: wiki/Offset/fr.md [hunk 13]
```diff
@@ -1,22 +1,27 @@
...
+- [Points de timing hérités](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) ("ligne verte"), une [timing section](/wiki/Beatmapping/Timing_section) qui contrôle des paramètres tels que le [kiai](/wiki/Gameplay/Kiai_time), le [sampleset](/wiki/Beatmapping/Sampleset), la [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), et le niveau de volume des hitsounds.
+- Beatmap offset, la quantité de temps entre le début du fichier audio et le premier [downbeat](/wiki/Music_theory/Downbeat) de la chanson, tel que contrôlé par les points de synchronisation.
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:19]:
```suggestion
- Beatmap offset, le temps entre le début du fichier audio et le premier [downbeat](/wiki/Music_theory/Downbeat) de la chanson, tel que contrôlé par les points de timing.
```

File: wiki/Offset/fr.md [hunk 14]
```diff
@@ -1,22 +1,27 @@
...
-- Un "offset" de beatmap est le temps entre le début du fichier audio et le premier [downbeat](/wiki/Music_theory/Downbeat) de la musique, mesuré en millisecondes.
-- [Online offset](/wiki/Offset/Online_offset) : changement de temps appliqué par un membre de la [NAT](/wiki/People/Nomination_Assessment_Team) aux beatmaps classées avec un offset incorrect non hérité afin de l'ajuster.
-- [Décalage local ("hors ligne")](/wiki/Offset/Local_offset), un paramètre par beatmap qui décale les objets dans le temps pendant le jeu.
-- [Décalage universel](/wiki/Offset/Universal_offset), le paramètre du client osu! qui affecte globalement le timing des beatmaps.
+## Paramètres de beatmap
 
-## Paramètres d'une beatmap
+*Note: Les décalages de l'introduction et de la vidéo ne sont modifiables que par l'[édition de fichier de beatmap](/wiki/Client/File_formats/Osu_(file_format)).*
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:23]:
```suggestion
*Note : Les décalages de l'introduction et de la vidéo ne sont modifiables que par l'[édition du fichier de la beatmap](/wiki/Client/File_formats/Osu_(file_format)).*
```

File: wiki/Offset/fr.md [hunk 15]
```diff
@@ -1,22 +1,27 @@
...
 
-- Décalage du compte à rebours, un paramètre qui affecte l'apparence du [compte à rebours](/wiki/Beatmap/Countdown) avant le premier objet.
-- Décalage audio, ou [entrée audio](/wiki/Beatmap/Lead-in_time).
-- Décalage vidéo, un paramètre qui indique à quel moment une vidéo de fond doit être lue par rapport au fichier audio.
+- offset de compte à rebours, un paramètre qui affecte le temps d'apparition du  [compte à rebours](/wiki/Beatmap/Countdown) avant le premier objet, mesuré en [beats](/wiki/Music_theory/Beat).
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:25]:
```suggestion
- Countdown offset, un paramètre qui affecte le temps d'apparition du [compte à rebours](/wiki/Beatmap/Countdown) avant le premier objet, mesuré en [beats](/wiki/Music_theory/Beat).
```

File: wiki/Offset/fr.md [hunk 16]
```diff
@@ -1,22 +1,27 @@
...
+- [Lead-in](/wiki/Beatmap/Lead-in_time), une période de retard supplémentaire avant que la beatmap ne commence.
+- Décalage vidéo, un paramètre qui détermine quand une vidéo d'arrière plan doit être lue par rapport à la piste audio.
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:27]:
```suggestion
- Video offset, un paramètre qui détermine quand une vidéo d'arrière plan doit être lue par rapport à la piste audio.
```

File: wiki/Offset/fr.md [hunk 17]
```diff
@@ -1,22 +1,27 @@
...
+## Paramètres de beatmap
 
-## Paramètres d'une beatmap
+*Note: Les décalages de l'introduction et de la vidéo ne sont modifiables que par l'[édition de fichier de beatmap](/wiki/Client/File_formats/Osu_(file_format)).*
 
-- Décalage du compte à rebours, un paramètre qui affecte l'apparence du [compte à rebours](/wiki/Beatmap/Countdown) avant le premier objet.
-- Décalage audio, ou [entrée audio](/wiki/Beatmap/Lead-in_time).
-- Décalage vidéo, un paramètre qui indique à quel moment une vidéo de fond doit être lue par rapport au fichier audio.
+- offset de compte à rebours, un paramètre qui affecte le temps d'apparition du  [compte à rebours](/wiki/Beatmap/Countdown) avant le premier objet, mesuré en [beats](/wiki/Music_theory/Beat).
+- [Lead-in](/wiki/Beatmap/Lead-in_time), une période de retard supplémentaire avant que la beatmap ne commence.
```
Comment (Ganondorf-1) [inline wiki/Offset/fr.md:26]:
```suggestion
- [Lead-in](/wiki/Beatmap/Lead-in_time), une période de délai supplémentaire avant que la beatmap ne commence.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**